fdbserver/GlobalTagThrottler.actor.cpp

```diff
@@ -465,10 +487,14 @@ class GlobalTagThrottlerImpl {
 		throttlingRatios[ss.id] = ss.getThrottlingRatio(SERVER_KNOBS->TARGET_BYTES_PER_STORAGE_SERVER,
```

Comment:
This function will update `throttlingRatios` even if `throughput` doesn't get updated below for any of the tags -- is that the desired behavior?

Reply:
Yeah, that is the desired behaviour, because we still want to track storage server health, even if there are no recent updates for tags on this storage server.